bilby/core/utils/random.py

```diff
@@ -11,7 +11,11 @@ class Generator:
 
 
 def seed(seed):
+    from .meta_data import global_meta_data
```

Comment:
I was running into circular imports but that may have been an old thing, I'll have a look

Reply:
Yeah, if I make this a top-level import, then I hit circular import issues. I think this is because everything is imported in the  `__init__.py` files in `bilby`, `bilby.core` and `bilby.core.utils`. We could make an exception for `meta_data` file.